Assets/_BForBoss/_Core/Scripts/UserInterface/Tutorial/TutorialCanvas.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using Perigon.Utility;
+using Sirenix.OdinInspector;
+using UICore;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace BForBoss
+{
+    public class TutorialCanvas : MonoBehaviour
+    {
+        [Title("Buttons")] 
+        [SerializeField] private Button _backButton;
+        [SerializeField] private Button _basicButton;
```

Comment:
It could easier for scaling to use a dictionary (and use Odin serialize it) of `<Button, CaroselView>` for all of the tutorials. So that we can just add more tutorials in the inspector in the future rather than copy/paste in the code.

And iterate over the values for the `ForEachTutorialCarouselView` method

Reply:
Because it's only like 4 - 5, was thinking it was fine.

Don't really wanted to add more if it's not going to change too much.